src/System.ComponentModel.Annotations/src/System/ComponentModel/DataAnnotations/FileExtensionsAttribute.cs

```diff
@@ -48,14 +48,7 @@ private IEnumerable<string> ExtensionsParsed
 
         private bool ValidateExtension(string fileName)
         {
-            try
-            {
-                return ExtensionsParsed.Contains(Path.GetExtension(fileName).ToLowerInvariant());
-            }
-            catch (ArgumentException)
-            {
-                return false;
-            }
+            return ExtensionsParsed.Contains(Path.GetExtension(fileName).ToLowerInvariant());
```

Comment:
`Path.GetExtension` throws on netfx if the path is invalid, but this doesn't happen anymore with netcore